config/routes.rb

```diff
@@ -5,4 +5,3 @@
   resources :posts, only: [:index, :show]
   end
 end
```

Comment:
- [ ] [optional] It would have been nice if your app had a root route, it makes it looks nices